github/orgs_custom_roles.go

```diff
@@ -44,3 +47,74 @@ func (s *OrganizationsService) ListCustomRepoRoles(ctx context.Context, org stri
 
 	return customRepoRoles, resp, nil
 }
+
+// Options required to create or update a custom repository role
+type CreateOrUpdateCustomRoleOptions struct {
+	Name        *string   `json:"name,omitempty"`
+	Description *string   `json:"description,omitempty"`
+	BaseRole    *string   `json:"base_role,omitempty"`
+	Permissions *[]string `json:"permissions,omitempty"`
+}
+
+// CreateCustomRepoRole creates a custom repository role in this organization
+// In order to create custom repository roles in an organization, the authenticated user must be an organization owner.
+//
+// GitHub API docs: https://docs.github.com/en/rest/orgs/custom-roles#create-a-custom-role
+func (s *OrganizationsService) CreateCustomRepoRole(ctx context.Context, org string, opts *CreateOrUpdateCustomRoleOptions) (*CustomRepoRoles, *Response, error) {
+	u := fmt.Sprintf("orgs/%v/custom_roles", org)
+
+	req, err := s.client.NewRequest("POST", u, opts)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	resultingRole := new(CustomRepoRoles)
+	resp, err := s.client.Do(ctx, req, resultingRole)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return resultingRole, resp, err
+}
+
+// CreateCustomRepoRole creates a custom repository role in this organization
+// In order to update custom repository roles in an organization, the authenticated user must be an organization owner.
+//
+// GitHub API docs: https://docs.github.com/en/rest/orgs/custom-roles#update-a-custom-role
+func (s *OrganizationsService) UpdateCustomRepoRole(ctx context.Context, org string, roleID string, opts *CreateOrUpdateCustomRoleOptions) (*CustomRepoRoles, *Response, error) {
+	u := fmt.Sprintf("orgs/%v/custom_roles/%v", org, roleId)
```

Comment:
This one is not going to compile. Please run unit tests locally before pushing. See CONTRIBUTING.md for more information.

```suggestion
	u := fmt.Sprintf("orgs/%v/custom_roles/%v", org, roleID)
```